CHANGELOG.md

```diff
@@ -15,6 +38,7 @@ These should not affect the functionality of the module.
 
 **Merged pull requests:**
 
+- \[chore\] release 10.0.1 [\#834](https://github.com/voxpupuli/puppet-rabbitmq/pull/834) ([wyardley](https://github.com/wyardley))
```

Comment:
I added `skip-changelog` to that PR. please regenerate the changelog.

Reply:
Thanks @bastelfreak 
saw it in there but forgot the right label to use. done.